src/component/child/index.js

```diff
@@ -235,6 +236,29 @@ export class ChildComponent extends BaseComponent {
         // TODO: Implement this
     }
 
+    watchForResize() {
+        let elm = document.body;
+        let lastWidth = elm.scrollWidth;
+        let newWidth;
+        let lastHeight = elm.scrollHeight;
+        let newHeight;
+
+        setInterval(() => {
+            newWidth = elm.scrollWidth;
+            newHeight = elm.scrollHeight;
+            // Dimensions changed if this condition is true
+            if (lastHeight !== newHeight || lastWidth !== newWidth) {
+                let newDimensions = {
+                    width: newWidth,
+                    height: newHeight
+                };
+                postRobot.sendToParent(POST_MESSAGE.RESIZE, newDimensions);
+            }
+            lastWidth = newWidth;
+            lastHeight = newHeight;
+        }, 10);
```

Comment:
Can we maybe make this 50ms? I'm concerned running this 100 times every second is going to be too much overhead.